test/e2e_node/cpu_manager_test.go

```diff
@@ -557,6 +559,92 @@ func runMultipleCPUContainersGuPod(ctx context.Context, f *framework.Framework)
 	waitForContainerRemoval(ctx, pod.Spec.Containers[1].Name, pod.Name, pod.Namespace)
 }
 
+func runCfsQuotaGuPods(ctx context.Context, f *framework.Framework, disabledCPUQuotaWithExclusiveCPUs bool) {
+	var err error
+	var ctnAttrs []ctnAttribute
+	var pod1, pod2, pod3 *v1.Pod
+	var cleanupPods []*v1.Pod
+	ginkgo.DeferCleanup(func() {
+		// waitForContainerRemoval takes "long" to complete; if we use the parent ctx we get a
+		// 'deadline expired' message and the cleanup aborts, which we don't want.
+		ctx2 := context.TODO()
+		ginkgo.By("by deleting the pods and waiting for container removal")
+		for _, cleanupPod := range cleanupPods {
+			framework.Logf("deleting pod: %s/%s", cleanupPod.Namespace, cleanupPod.Name)
+			deletePodSyncByName(ctx2, f, cleanupPod.Name)
+			waitForContainerRemoval(ctx2, cleanupPod.Spec.Containers[0].Name, cleanupPod.Name, cleanupPod.Namespace)
+			framework.Logf("deleted pod: %s/%s", cleanupPod.Namespace, cleanupPod.Name)
+		}
+	})
+
+	cfsCheckCommand := []string{"sh", "-c", "cat /sys/fs/cgroup/cpu.max && sleep 1d"}
+	defaultPeriod := "100000"
+
+	ctnAttrs = []ctnAttribute{
+		{
+			ctnName:    "gu-container-cfsquota-disabled",
+			cpuRequest: "1",
+			cpuLimit:   "1",
```

Comment:
Can you add another test with two containers here something like this;
```
ctnAttrs = []ctnAttribute{
		{
			ctnName:    "gu-container-int-values",
			cpuRequest: "1",
			cpuLimit:   "1",
		},
		{
			ctnName:    "gu-container-non-int-values",
			cpuRequest: "20m",
			cpuLimit:   "20m",
		},
	}
```

Reply:
done!